src/bullsAndCows.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const readline = require('readline');
+const { getRandomNumber } = require('./getRandomNumber');
+const { countBulls } = require('./countBulls');
+const { countCows } = require('./countCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const computerNumber = getRandomNumber();
+
+function handlePlayerNumber(input) {
+  const [bulls, cows] = [
+    countBulls(input, computerNumber),
+    countCows(input, computerNumber),
```

Comment:
looks great but I am not sure those two have to be separate functions

Reply:
agree, you can avoid going throw input value 2 times and create one function for calculating both of values